ui/app/scripts/directives.js

```diff
@@ -393,5 +393,40 @@ define(['angular', 'xregexp', 'moment'], function(angular) {
           });
         }
       };
+    })
+    .directive('tableSort', function() {
+      function applySort(scope) {
+        if( scope.sortState.sortProperty.toString() === scope.sortProperty.toString() && scope.sortState.sortOrder === 'DESC' ) {
```

Comment:
Why are `scope.sortState.sortProperty` and `scope.sortProperty` being checked here? For the given page, these properties are static, isn't it?